jsonnet/cluster-monitoring-operator.jsonnet

```diff
@@ -0,0 +1,48 @@
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+
+{
+  _config+:: {
+    clusterMonitoringOperatorSelector: 'job="cluster-monitoring-operator"',
+    jobs+: {
+      ClusterMonitoringOperator: $._config.clusterMonitoringOperatorSelector,
```

Comment:
@mxinden the absent job alert is created thanks to this field

Reply:
Cool, thanks for clarifying @squat!